pages/bootcamps/bootcamp25-announce.md

```diff
@@ -0,0 +1,79 @@
+This includes graduate students, postdoctoral researchers, early career professionals, and many others.
+The ideal participants are those who, after some hands-on experience writing code, are looking for more software development training to either prepare them for a career writing research software or to be a more efficient developer.
+In either case, attendees should have a strong interest in developing research software in the future.
+
+We expect attendees to come with a basic background in programming.
+Previous, formal computer science (CS) training is specifically not a prerequisite.
+Rather, we expect many, if not most attendees to be self-taught programmers coming from non-CS domains.
+Where possible, we will aim to keep instruction uncoupled from specific languages or technologies.
+Because this is nearly impossible, we expect attendees to have a working knowledge of python, basic git commands and functionality, competency with an editor, and experience working with the command line.
+Additionally, participants should be comfortable managing their own development environment on their laptop.
+
+Our INTERSECT [learner profiles]({{ site.url }}{{ site.baseurl }}/learner-profiles/) provide examples of the types of people for whom this workshop is appropriate.
+However, we welcome anyone who fits the description above to submit an application for the Bootcamp.
+
+## Topics/Agenda
+The bootcamp will run approximately 8:30-4 Monday through Thursday and 8:30-12:00 on Friday.
+To provide a break during the week, the bootcamp will only meet until lunch on Wednesday.
+
+The bootcamp will focus on the following topics:
+* Software Design
+* Collaborative git
+* Pull Requests
+* Code review
+* Licensing
+* Documentation
+* Testing
+* CI/CD
+* Packaging & Distribution
+
+Each session will include lectures along with hands-on activities and exercises, including some opportunities to work in groups. Participants will be expected to bring their own laptop.
+
+Participants are expected to attend the entirety of the bootcamp.
+Failure to do so may result in a forfeiture of all or part of a travel grant.
+
+## Cost & Funding
+The INTERSECT project, through a grant from the National Science Foundation, has funding to cover the food, travel, and lodging expenses for up to 30 attendees as a travel grant.
+This travel grant includes 5 nights of hotel plus up to $800 for transportation and food (reimbursed via Princeton University or the University of Alabama).
+In addition to those funded directly by the INTERSECT project, we will accept self-funded participants as space allows.
+
+During the workshop we will provide breakfast and lunch (M-F) for all participants (whether supported by INTERSECT or paying their own way).
+There will be two dinner receptions on Monday and Tuesday evening sponsored by Princeton Research Computing. 
```

Comment:
```suggestion
```
Let's leave it out for now as I haven't secured the funding yet (I suspect it will be fine, I just need to formally request)